chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Task 1 - existance of files
+set -eo pipefail
+
+
+# Error handling function.
+err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+}
+
+
+# Function for delimiter
+print_separator() {
+  printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+}
+
+
+# Function to generate a random string and write to a file with permissions
+create_file() {
+  local filecr="$1"
```

Comment:
That is rather strange name for variable `filecr`, I think just `file` would be ok